File: Ser2023/Company/Company.Web/Modules/Company/ContactType/ContactTypeForm.cs
using Serenity.ComponentModel;
using System;

namespace Company.Company.Forms;

[FormScript("Company.ContactType")]
[BasedOnRow(typeof(ContactTypeRow), CheckNames = true)]
public class ContactTypeForm
{
    public string Name { get; set; }
    //public Guid Rowguid { get; set; }
    //public short IsActive { get; set; }
    //public int TenantId { get; set; }
}

File: Ser2025/Company/Company.Web/Modules/ServerTypes/Company/ContactTypeForm.ts
import { PrefixedContext, IntegerEditor, StringEditor, initFormType } from '@serenity-is/corelib';

export interface ContactTypeForm {
    Name: StringEditor;
    IsActive: IntegerEditor;
    TenantId: IntegerEditor;
}

export class ContactTypeForm extends PrefixedContext {
    static readonly formKey = 'Company.ContactType';
    private static init: boolean;
    
    constructor(prefix: string) {
        super(prefix);
        if (!ContactTypeForm.init)  {
            ContactTypeForm.init = true;
            
            var w0 = IntegerEditor;
            var w1 = StringEditor;

            initFormType(ContactTypeForm, [
            'Name', w1,
            'IsActive', w0,
            'TenantId', w0,
            ]);
        }
    }
}